recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
Also, to clarify - the lack of support for shared fortran libs on OSX goes back to at least 1.8.14.